view/frontend/web/template/magic/minicart.html

```diff
@@ -0,0 +1,98 @@
+<!--
```

Comment:
1. Override in Payment Method module is really bad idea,
2. Additionally -- You should not include outdated Magento copyright message.